share/models/ingest.py

```diff
@@ -126,6 +126,8 @@ class SourceConfig(models.Model):
 
     disabled = models.BooleanField(default=False)
 
+    json_encrypt = EncryptedJSONField(blank=True, null=True)
```

Comment:
Could you replace the `transformer_kwargs` and `harvester_kwargs` with the encrypted field?
You'll have to write a migration for it. I think the best way would be:
* Add a new encrypted column/field with different names
* Iterate over all the rows and copy the settings over, in python
* Drop the old columns
* Rename the new ones